src/lib/FeatureServices.js

```diff
@@ -4,6 +4,7 @@
 
 var fs = require('fs')
 var path = require('path')
+var moment = require('moment')
```

Comment:
no need to make a change for this PR, but just a note to switch to using let and const on the 3.0 branch.